bitcoin/src/address/mod.rs

```diff
@@ -140,7 +140,7 @@ pub trait NetworkValidation: sealed::NetworkValidation + Sync + Send + Sized + U
 /// This allows users to use `V: NetworkValidation` in conjunction with derives. Is only ever
 /// implemented for `NetworkUnchecked`.
 pub trait NetworkValidationUnchecked:
-    NetworkValidation + sealed::NetworkValidationUnchecked + Sync + Send + Sized + Unpin
```

Comment:
These traits might affect the traits that `Address` has, not anything to do with `Serialize`. We might have to implement them manually.